work_test.go

```diff
@@ -238,21 +239,19 @@ func TestLockJobAdvisoryRace(t *testing.T) {
 	getBackendID := func(conn *pgx.Conn) int32 {
 		var backendID int32
 		err := conn.QueryRow(`
-			SELECT backendid
-			FROM pg_stat_get_backend_idset() psgb(backendid)
-			WHERE pg_stat_get_backend_pid(psgb.backendid) = pg_backend_pid()
+			SELECT pg_backend_pid()
 		`).Scan(&backendID)
 		if err != nil {
 			panic(err)
 		}
 		return backendID
 	}
-	waitUntilBackendIsWaiting := func (backendID int32, name string) {
+	waitUntilBackendIsWaiting := func(backendID int32, name string) {
 		conn := newConn()
 		i := 0
 		for {
 			var waiting bool
-			err := conn.QueryRow(`SELECT pg_stat_get_backend_waiting($1)`, backendID).Scan(&waiting)
```

Comment:
yeah, looks like it was removed. Compare the [9.5 docs](https://www.postgresql.org/docs/9.5/static/monitoring-stats.html#MONITORING-STATS-BACKEND-FUNCS-TABLE) to the [9.6 docs](https://www.postgresql.org/docs/9.6/static/monitoring-stats.html#MONITORING-STATS-BACKEND-FUNCS-TABLE).

Can you briefly explain how these are equivalent?